tests/tacacs/utils.py

```diff
@@ -274,6 +277,14 @@ def remove_all_tacacs_server(duthost):
             duthost.shell("sudo config tacacs delete %s" % tacacs_server)
 
 
+def count_authorization_request(ptfhost):
+    hex_string = binascii.hexlify("cmd=/".encode('ascii')).decode()
+    sed_command = "sed -n 's/.*-> 0x\(..\).*/\\1/p'  /var/log/tac_plus.log | sed ':a; N; $!ba; s/\\n//g'"  # noqa W605 E501
```

Comment:
>W605 [](http://example.com/codeflow?start=114&length=4)

Is it possible to fix this warning? any invalid escape sequence in code?

Reply:
There is no invalid escape sequence, this is a false alert by flake8